subsys/nrf_security/src/drivers/cracen/cracenpsa/src/platform_keys/platform_keys.c

```diff
@@ -699,10 +712,44 @@ psa_status_t cracen_platform_keys_provision(const psa_key_attributes_t *attribut
 	uint32_t sicr_attr = (key.sicr.bits << 16) | key.sicr.type;
 
 	if (key.sicr.type == PSA_KEY_TYPE_AES) {
-		write_sicr_key_to_mram(key, sicr_attr, encrypted_key, key_buffer_size);
+		write_sicr_key_to_mram(&key, sicr_attr, encrypted_key, key_buffer_size);
 	} else {
-		write_sicr_key_to_mram(key, sicr_attr, key_buffer, key_buffer_size);
+		write_sicr_key_to_mram(&key, sicr_attr, key_buffer, key_buffer_size);
 	}
 
 	return status;
 }
+
+psa_status_t cracen_platform_destroy_key(const psa_key_attributes_t *attributes)
+{
+	platform_key key;
+	key_type type = find_key(MBEDTLS_SVC_KEY_ID_GET_KEY_ID(psa_get_key_id(attributes)), &key);
+	/* The value 0x00 was chosen arbitrarily here, 0xFF was not used to distinguish revoked keys
+	 * from keys not yet written.
+	 */
+	const static uint8_t revoked_key_val[MAX_KEY_SIZE] = {0x0};
```

Comment:
typically rather
```suggestion
	static const uint8_t revoked_key_val[MAX_KEY_SIZE] = {0x0};
```
(2344 vs 55 hits in `sdk-nrf`)